test/controllers/order_items_controller_test.rb

```diff
@@ -0,0 +1,69 @@
+require "test_helper"
+
+describe OrderItemsController do
+  let(:user) { users(:beyonce) }
+
+  describe "index" do
+    it "redirects to root path if user is not a merchant" do
+      user = nil
```

Comment:
this line just makes a local variable named `user` and assigns the value to it as `nil`-- this doesn't affect the app at all

Think about what you're trying to test here and how you would make the "user" in the app not-exist... or "logged out"